sway-fmt/src/fmt.rs

```diff
@@ -641,4 +641,21 @@ fn one_liner() -> bool {
         let (_, formatted_code) = result.unwrap();
         assert_eq!(correct_sway_code, formatted_code);
     }
+
+    #[test]
+    // Test that the use statements with multiple imports are properly formatted
+    fn test_use_statement() {
+        let expected_sway = r#"script;
+use std::chain::{panic,log_u8};
+use std::chain::assert;
+use std::hash::{HashMethod,hash_value,hash_pair};
```

Comment:
Will you please double check that more complex patterns are also fixed with your change? Example:
```rust
use a::b::{c, d::{e, f}};
use a::b::{c, d::{self, f}};
```

Reply:
Ouch, I can try hahah